chronos/internal/asyncmacro.nim

```diff
@@ -219,7 +219,8 @@ proc decodeParams(params: NimNode): AsyncParams =
   var
     raw = false
     raises: NimNode = nil
-    handleException = chronosHandleException
+    handleException = false
+    handleExceptionOverridden = false
```

Comment:
I tried to avoid using this, but I couldn't think of a way to otherwise distinguish a local override where the annotation in the proc sets `handleException` to false (which you shouldn't override with the global flag) vs when nothing has been set and you're just looking at the initialization value (in which case you should override with the global flag).